pkgs/desktops/xfce/applications/xfmpc/default.nix

```diff
@@ -0,0 +1,58 @@
+{ lib
+, stdenv
+, fetchFromGitLab
+, autoreconfHook
+, xfce4-dev-tools
+, libxfce4ui
+, libxfce4util
+, pkg-config
+, vala
+, gtk3
+, glib
+, libmpd
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "xfmpc";
+  # Last release is too old
+  version = "0.3.1-2024-05-29";
+
+  src = fetchFromGitLab {
+    domain = "gitlab.xfce.org";
+    owner = "apps";
+    repo = "xfmpc";
+    rev = "cf40dffec6e9b80abb1f1aa6d7dceef4790173dc";
+    hash = "sha256-moCWSLGBJuWM4/lRJi6D3w38iJeCntLo3Vl/eVfu7lw=";
+  };
+  nativeBuildInputs = [
+    autoreconfHook
+    xfce4-dev-tools
+    pkg-config
+    vala
+    # Needed both here and in buildInputs for cross compilation to work
+    libxfce4ui
+    libxfce4util
```

Comment:
Ugh, it looks like the cross `libxfce4{ui,util}` build failed to find native `vapigen` and thus not building the `*.vapi` files.